concrete/bootstrap/helpers.php

```diff
@@ -333,6 +339,7 @@ function core_log($message,
  * @param  string  $abstract
  * @param  array   $parameters
  * @return mixed
+ * @since 5.7.0 (but not in 8.3.0 8.3.1 8.3.2 8.4.0 8.4.1 8.4.2 8.4.3 8.4.4 8.4.5 8.5.0 8.5.1)
```

Comment:
@mlocati This should be `@since 8.5.2`

Reply:
Well, before 8.3.0, the `app()` global function was defined by [`illuminate/support`](https://github.com/illuminate/support/blob/v4.1.30/helpers.php#L26).
Since 8.5.2, concrete5 has its own implementation of it....

Reply:
I see, but it's confusing.

Reply:
I don't know why this function does not exist in 8.3.0+, but how about to change like this?

```
@since 5.7.0 defined by laravel but not in concrete5 core
@since 8.3.0 removed in laravel
@since 8.5.2 re-implemented. See #8011
```